django_stripe/management/commands/charge_stripe.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+import stripe
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from django_stripe.models import StripeCharge
+
+
+class Command(BaseCommand):
+    help = "Charge stripe"
+
+    def handle(self, *args, **options):
+        charges = StripeCharge.objects.filter(is_charged=False)
+        stripe.api_key = settings.STRIPE_API_KEY
+        for c in charges:
+            if c.token.is_active:
+                stripe_charge = stripe.Charge.create(
```

Comment:
You mean catch exceptions? We don't do it on purpose here - if it fails it will be retried. And yes, we do have some errors like problems with card being expired at the moment of charge. I think this should not block release and we will fix this as we go. On the other hand, if the error happens here it should not be silenced . THe only place error will happen is this create command, and if it breaks system is not altered in any way so... would that not be an overkill? 

@poxip please add a TODO section to readme and add there notes about this and other one.